native/cocos/bindings/event/EventDispatcher.cpp

```diff
@@ -212,6 +214,12 @@ void EventDispatcher::dispatchMouseEvent(const MouseEvent &mouseEvent) {
         }
         jsMouseEventObj->setProperty("x", xVal);
         jsMouseEventObj->setProperty("y", yVal);
+        if (type == MouseEvent::Type::MOVE) {
+            const auto &xrefVal = se::Value(mouseEvent.xref);
+            const auto &yrefVal = se::Value(mouseEvent.yref);
+            jsMouseEventObj->setProperty("xref", xrefVal);
+            jsMouseEventObj->setProperty("yref", yrefVal);
+        }
```

Comment:
I think only pointlock mode need to pass delta value.

Reply:
We should pass as less data as possible.

Reply:
If the event is a move click, it will cause a coordinate exception

Reply:
What i mean is that, should pass different data to TS in different mode. Passing delta value in not pointerlock mode is meaningless.

Reply:
I think it would be better to be consistent. In fact, on the ts is required to calculate the delta data, if passed up through the native, we do not need to calculate

Reply:
As passing more data will affect performance. That's why we should pass as less data as possible, especially the move event.

Reply:
As passing more data will affect performance. That's why we should pass as less data as possible, especially the move event.

Reply:
Did you mean delta value is needed in non pointerlock mode too?

Reply:
Yes, it is need to calculate.
![image](https://user-images.githubusercontent.com/5205637/218920403-fa2d7a2f-1c2b-4ad1-b91b-861f87690e68.png)